source/git-util.js

```diff
@@ -117,9 +117,9 @@ const gitVersion = async () => {
 
 exports.verifyRecentGitVersion = async () => {
 	const installedVersion = await gitVersion();
-	const minVersion = '2.11.0';
+	const range = require('../package.json').engines.git;
```

Comment:
This should be extracted to a separate utility method, named something like `getRequiredVersion(pkg, requirementName)` (i.e. `getRequiredVersion(pkg, 'git')`), to make the functionality reusable.
Another option is to add `satisfiesRequiredVersion(requirementName, version)` (i.e. `satisfiesRequiredVersion('git', '2.12.0')`), which might be better in this case.